ansible/roles/runner/handlers/main.yml

```diff
@@ -10,11 +10,15 @@
   poll: 0
   ignore_errors: true
   notify:
-    - wait for machine
+    - reboot_message
 
 - name: wait for machine
```

Comment:
Is this handler used from somewhere else? If not, why do you keep it here?

Reply:
It's not used anywhere else. It's just a quick way to run multiple tasks when I trigger a handler. Do you see a better place for the code?